approval_polls/static/approval_polls/detail.js

```diff
@@ -1,4 +1,7 @@
-$(function () {  
+$(function () {
+  $("#share").jsSocials({
+    shares: ["twitter", "facebook", "googleplus", "linkedin", "pinterest"]
```

Comment:
@CCInCharge, could you also add 'email' and keep only the icons on the text. I think the word Like and G+1 on the buttons look misleading as the title of a share button. Let me know if you think otherwise.